### DISCUSSION TITLE
Will Tesla Android support Raspberry Pi 5?

### DISCUSSION THREAD

**Discussion by 8bitjer** [tesla-android/issue-tracker#301] [General]
I was going to pre-order with the thought that Carplay would run smoother and faster at higher resolutions. Are there any plans to support the 5 when it is released?

**Comment by mikegapinski** [comment]
No, the Pi 5 needs too much power and outputs too much heat for my application. I'll make the final decision once I get my hands on the compute module 5, maybe this thing would be manageable.
